Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -59,6 +60,7 @@ lemma traceForm_comm (x y : L) : traceForm R L M x y = traceForm R L M y x :=
 @[simp] lemma traceForm_flip : (traceForm R L M).flip = traceForm R L M :=
   Eq.symm <| LinearMap.ext₂ <| traceForm_comm R L M
 
+/-- See also `LieModule.traceForm_apply_lie_apply'`. -/
```

Comment:
A docstring should first explain the content of the lemma. Once this is done, it's fine to add references to other lemmas, but without that the docstring would be weird when hovering over the lemma.

Reply:
Done!